media/css/mozorg/contribute/contribute-2015.less

```diff
@@ -154,6 +154,12 @@ textarea:focus {
     }
 }
 
+.select.invalid {
+    border-color: #a91300;
+    @shadow: 0 0 0 2px rgba(255,80,80,0.4);
```

Comment:
Any need for a variable here? Could just use `.box-shadow(0 0 0 2px rgba(255, 80, 80, 0.4));`?